### PR TITLE
More tests and fix of bugs

### PR DESCRIPTION
**Pull request by DanielAndreasen**
This PR fix bug #48 and comes with small improvements.

### CODE REVIEW

File: sweetercat/plot.py [hunk 1]
```diff
@@ -194,6 +197,8 @@ def plot_page(df, columns, request, page):
     css_resources = INLINE.render_css()
 
     script, div = components(layout)
+    if error is not None:
```
Comment (jason-neal) [inline sweetercat/plot.py]: If you changed the error to be `True / False` only then you would only need `if error:`

Any reason for `True / None`?

Comment (DanielAndreasen) [inline sweetercat/plot.py]: No reason. Just what I'm used to write :)

**Comment by jason-neal** [comment]
I think it's time to merge this PR. For the lines untested I am not sure if they can be. 
Should probably merge tests and then handle errors in the clean plot code.

**Comment by DanielAndreasen** [comment]
I think we should not worry about the few lines of code in `app.py` that is not tested. One is to run the server, which we know works, another is to safely remove a file.

**Comment by jason-neal** [comment]
Yes I do agree that we don't need to worry with them.

**Pull request merged**